docs/reference/api/pages.md

```diff
@@ -307,7 +309,7 @@ const data = {
   _position: 'lastChild'
 };
 // Request inside an async function.
-const response = await fetch('http://example.net/api/v1/@apostrophecms/page/ckitdo5oq004pu69kr6oxo6fr:en:published?apikey=myapikey', {
+const response = await fetch('http://example.net/api/v1/@apostrophecms/page/ckitdo5oq004pu69kr6oxo6fr?apikey=myapikey&aposMode=published&aposLocale=fr', {
```

Comment:
If you are going to show examples using `aposDocId` instead of the complete `_id` you need to show that so developers know where to get it from.

Also we don't want to lose all examples showing it's perfectly fine to use the plain `_id` if you're happy with the locale and mode in it.

Reply:
I changed a few back and expanded upon the note within the topmost GET that takes `_id`. Wasn't sure if it would be redundant to add that same message in multiple areas.